cpp/test/model_test/logistic_regression_benchmark.cc

```diff
@@ -43,12 +43,16 @@ class Benchmark: public ::testing::Test {
   Nice::Vector<T> expected_vals;
```

Comment:
Please move your benchmark tests to a directory named performance_test, so this file's path would be something like:

cpp/test/performance_test/logistic_regression_performance

Reply:
Because we have decided that performance test should be in its own branch, please scratch the last comment, just move everything performance related to your performance branch.

Reply:
Fixed, see new commit.